habitat-lab/habitat/config/CONFIG_KEYS.md

```diff
@@ -0,0 +1,180 @@
+| habitat.task.lab_sensors.is_holding_sensor | Rearrangement only. A single float sensor with value 1.0 if the robot is grasping any object and 0.0 otherwise.|
+|habitat.task.lab_sensors.end_effector_sensor | Rearrangement only. the cartesian coordinates (3 floats) of the arm's end effector in the frame of reference of the robot's base.|
+|habitat.task.lab_sensors.joint_sensor| Rearrangement only. Returns the joint positions of the robot.|
+| habitat.task.lab_sensors.goal_sensor |     Rearrangement only. Returns the relative position from end effector to a goal position in which the agent needs to place an object. |
+| habitat.task.lab_sensors.target_start_gps_compass_sensor |     Rearrangement only. Returns the initial position of every object that needs to be rearranged in composite tasks, in 2D polar coordinates. |
+| habitat.task.lab_sensors.target_goal_gps_compass_sensor |    Rearrangement only. Returns the desired goal position of every object that needs to be rearranged in composite tasks, in 2D polar coordinates. |
+
+## Rearrangement Measures
+| Key | Description |
+| --- | --- |
+|habitat.task.measurements.end_effector_to_rest_distance | Rearrangement only. Distance between current end effector position  and the resting position of the end effector. Requires that the   RelativeRestingPositionSensor is attached to the agent. |
+|habitat.task.measurements.robot_force |      The amount of force in newton's applied by the robot. It computes both the instant and accumulated. |
+|habitat.task.measurements.does_want_terminate | Rearrangement Only. Measures 1 if the agent has called the stop action and 0 otherwise.    |
+|habitat.task.measurements.force_terminate |    If the force is greater than a certain threshold, this measure will be 1.0 and 0.0 otherwise.   Note that if the measure is 1.0, the task will end as a result. |
+|habitat.task.measurements.force_terminate.max_accum_force |  The threshold for the accumulated force. -1 is no threshold.   |
+|habitat.task.measurements.force_terminate.max_instant_force |  The threshold for the current, instant force. -1 is no threshold.   |
+|habitat.task.measurements.object_to_goal_distance |  In rearrangement only. The distance between the target object and the goal position for the object. |
+|habitat.task.measurements.obj_at_goal |  The measure is a dictionary of target indexes to float. The values are 1 if the object is within succ_thresh of the goal position for that object.   |
+|habitat.task.measurements.obj_at_goal.succ_thresh | The threshold distance below which an object is considered at the goal location.    |
+|habitat.task.measurements.art_obj_at_desired_state |   Rearrangement open/close container tasks only. Whether the articulated object (fridge or cabinet door) towards a desired state (open or closed) as defined by the task. |
```

Comment:
```suggestion
|habitat.task.measurements.art_obj_at_desired_state |   Rearrangement open/close container tasks only. Whether the articulated object (fridge or cabinet door) is in a desired state (open or closed) as defined by the task. |
```

Reply:
How is open/close measured? based on articulated joint angles?

Reply:
There is a target articulated object joint state in the episode. The code seems to indicate this is based of joint state.